pennylane/shadows/classical_shadow.py

```diff
@@ -316,22 +317,139 @@ def qnode(x):
+                In case of an integer, a random subset of that size is taken. The subset can also be explicitly fixed by passing an Iterable with the corresponding indices.
+            alpha (float): order of the Renyi-entropy. Defaults to ``alpha=2``, which corresponds to the purity of the reduced state. This case is straight forward to compute.
+                All other cases ``alpha!=2`` necessitate computing the eigenvalues of the reduced state and thus may lead to longer computations times.
+                Another special case is ``alpha=1``, which corresponds to the von Neumann entropy.
+            k (int): Allow to split the snapshots into ``k`` equal parts and estimate the snapshots in a median of means fashion. There is no known advantage to do this for entropies.
+                Thus, ``k=1`` is default and advised.
+            base (float): Base to the logarithm used for the entropies.
+            atol (float): Absolute tolerance for Eigenvalues close to 0 that are taken into account.
+
+        Returns:
+            float: Entropy of the chosen subsystem.
+
+        **Example**
+
+        For the maximally entangled state of ``n`` qubits, the reduced state has two constant eigenvalues :math:`\frac{1}{2}`. For constant distributions, all Renyi entropies are
+        equivalent:
+
+        .. code-block:: python3
+
+            wires=4
+
+            @qml.qnode(dev)
+            def max_entangled_circuit():
+                qml.Hadamard(wires=0)
+                for i in range(1, wires):
+                    qml.CNOT(wires=[0, i])
+                return qml.classical_shadow(wires=range(wires))
+
+            bits, recipes = max_entangled_circuit()
+            shadow = ClassicalShadow(bits, recipes)
+
+            entropies = [shadow.entropy(wires=[0], alpha=alpha, atol=1e-2) for alpha in [1., 2., 3.]]
+
+        >>> np.isclose(entropies, entropies[0], atol=1e-2)
+        [ True,  True,  True]
+
+        For non-uniform reduced states that is not the case anymore and the entropy differs for each order ``alpha``:
+
+        .. code-block:: python3
+
+            @qml.qnode(dev)
+            def qnode(x):
+                for i in range(n_wires):
+                    qml.RY(x[i], wires=i)
+
+                for i in range(n_wires-1):
+                    qml.CNOT((i,i+1))
+
+                return classical_shadow(wires=dev.wires)
+
+            bitstrings, recipes = qnode(x)
+            shadow = ClassicalShadow(bitstrings, recipes)
+
+            entropies = [shadow.entropy(wires=wires, alpha=alpha, atol=1e-10) for alpha in [1., 2., 3.]]
+
+        >>> print(entropies)
+        [0.6727522114759635, 0.6252047673044356, 0.5996592216299593]
+
+        """
+        global_snapshots = self.global_snapshots(wires=wires, snapshots=snapshots)
+        rdm = median_of_means(global_snapshots, k, axis=0)
+
+        # Allow for different log base
+        if base:
+            div = np.log(base)
+        else:
+            div = 1
```

Comment:
Minor: more compact this way.
```suggestion
        div = np.log(base) if base else 1
```